src/newrelic.coffee

```diff
@@ -52,6 +64,18 @@ plugin = (robot) ->
   config.up = ':white_check_mark:'
   config.down = ':no_entry_sign:'
 
+  infraQuery = _.template("""
+    SELECT
+      average(cpuPercent) AS cpuPercent,
+      average(memoryUsedBytes / memoryTotalBytes) * 100 AS memoryPercent,
+      average(diskUsedPercent) as diskUsedPercent
```

Comment:
I have a humble request to include memory used as a counting integer of MBs as well as percent.

Reply:
@champain here I was just attempting to match (to the extent possible) the existing `servers` output, which only has percentages; might be a separate PR to expand this command's output?